### ISSUE TITLE
Induction Matrix doesnt form when internal components are below the input and output ports

### ISSUE THREAD

**Issue by v-tron** [mekanism/Mekanism#8163]
### Issue description

When building the induction matrix, the wiki says that internal components can be placed anywhere in the matrix, along with air blocks. However, if components are placed below the port level, the matrix doesnt form for some reason. I have not tested this extensively, however, if any internal components were placed below the ports (i had air blocks in between the components, i.e. induction providers on bottom layer and induction cells on top or vice versa) the matrix wouldn't form.

### Steps to reproduce

1. Build Induction Matrix shell
2. Put ports somewhere higher than the lowest level
3. Put internal components below ports and air above
4. Matrix doesn't form

### Minecraft version

1.20.x or earlier (No longer being developed)

### NeoForge version

Forge 47.2.20

### Mekanism version

Older

### Other relevant versions

Mekanism 10.4.8.43

**Comment by v-tron** [comment]
I have now tried testing this a little further and the behaviour is pretty much random it feels like. The matrix sometimes forms and sometimes doesn't. Debug logs say nothing. I will see if I can somehow make sense of this behaviour but I guess the issue can be closed for now.

**Comment by thiakil** [comment]
sounds like you have another mod placing not-actually-air blocks. Try right clicking an edge using a configurator in configure mode. It should show where an invalid block is, provided the frame is complete

**Comment by UnknownSerhan** [comment]
Does it fix itself when you break any block and place it back? Because that's a know bug in the mod.

**Comment by v-tron** [comment]
Yes it does, it seems to form as soon as I break a block and replace it

**Comment by UnknownSerhan** [comment]
Okay then this has to be the same issue as https://github.com/mekanism/Mekanism/issues/7621
